### ISSUE TITLE
iOS version sets screen.orientation to "locked" or "unlocked" instead of e.g. "portrait"

### ISSUE THREAD

**Issue by S-unya** [gbenvenuti/cordova-plugin-screen-orientation#88]
I am using cordova-plugin-screen-orientation v1.4.0, I have noticed that, while the Android version of the plugin works and reports the correct values, the iOS version sets screen.orientation to "locked" or "unlocked" once one calls the screen.setOrientation function.

**Steps:**

1.  Using Safari console, log `screen.orientation` => "portrait" as expected.
2. call `screen.unlockOrientation()` => undefined
3. log `screen.orientation` => "unlocked"
4. call `screen.lockOrientation('portait')` => undefined
5. log `screen.orientation` => "locked"

I see the aim, but I'm uncertain why it is eturning "unlocked" instead of an orientation value like "landscape". My Objective C is rusty, but it looks like that should be set to the orientation in there...

**Comment by S-unya** [comment]
I downgraded the cordova ios platform to @3.9.2 and this plugin begins working properly again on iOS. This may be to do with something I noticed while trying to fix platform 4.0.0; The Cordova method `shouldAutorotateToInterfaceOrientation` is never called, which I understood was the basis of this plugin.

I don't have time to continue looking now, but I will investigate further once this project is out of the way if you like.
